assets/section-footer.css

```diff
@@ -302,12 +301,34 @@
 }
 
 .localization-form__select {
-  box-shadow: 0 0 0 0.1rem rgba(var(--color-foreground), 0.55);
+  border-radius: var(--inputs-radius-outset);
   position: relative;
   margin-bottom: 1.5rem;
-  border: 0.1rem solid transparent;
   padding-left: 1rem;
   text-align: left;
+  min-height: calc(4rem + var(--inputs-border-width) * 2);
+  min-width: calc(7rem + var(--inputs-border-width) * 2);
+}
+
+.disclosure__button.localization-form__select {
+  padding: calc(1.5rem + var(--inputs-border-width))
+}
+
+.localization-form__select:before {
+  content: '';
+  position: absolute;
+  inset: 0;
+  border-radius: var(--inputs-radius-outset);
+  box-shadow: var(--inputs-shadow-horizontal-offset) var(--inputs-shadow-vertical-offset) var(--inputs-shadow-blur-radius) rgba(var(--color-base-text), var(--inputs-shadow-opacity));
+}
+
+.localization-form__select:after {
+  content: '';
+  position: absolute;
+  inset: var(--inputs-border-width);
+  border: 0.1rem solid transparent;
+  border-radius: var(--inputs-radius);
+  box-shadow: 0 0 0 var(--inputs-border-width) rgba(var(--color-foreground),var(--inputs-border-opacity)); 
```

Comment:
When using the max value for horizontal offset (negative value), the shadow is coming up over the previous summary. 

Not a big issue, just noting it.
![](https://screenshot.click/22-01-ngatg-dl182.png)

Reply:
I didnt have z index in the `after` and `before` and that his was done for the button PR. I added that and it worked great for all inputs except for the newsletter one (I had to explicitly set the z-index to 0 for the button) and the overlapping is still happening for the localization selectors 🤔 . 

https://github.com/Shopify/dawn/pull/1097/commits/226ef8f74e49019a8c6e664133ac8c9b68c1d24b 

![11-34-mqkl6-xz1ex](https://user-images.githubusercontent.com/19962891/149015524-3d0a82b9-1afb-4bdd-9158-84aa8b0428f1.gif)

Reply:
This is tricky so let me know if you want to pair.. Setting that z-index: 0 on the button does break the relative z context and will prevent this approach from working, but obviously, if you do that, the shadows disappear behind the footer bg. If you were to add a `z-index: 1` and `position relative` to some div that is a child of `.footer.color-background-1` but still a parent of the localization forms, it should fix that. Though the buttons seem to have `background: transparent` too so the shadow will still appear show through. So to fix _that_ we can change that transparent to `var(--color-background)` or whatever the footer gets.